src/mono/wasm/debugger/BrowserDebugProxy/JObjectValueCreator.cs

```diff
@@ -293,6 +293,21 @@ private async Task<JObject> ReadAsObjectValue(MonoBinaryReader retDebuggerCmdRea
                 return Create(value: className, type: "symbol", description: className);
             }
         }
+        var typeInfo = await _sdbAgent.GetTypeInfo(type_id[0], token);
```

Comment:
If a type has `ToString` overridden, and has a Debugger* attribute, then which one would be used?

Also, this won't work for ValueTypes, right?